index.html

```diff
@@ -53,7 +70,7 @@ <h3 class="masthead-brand">Hire Megan Black!</h3>
             <div style="width:15em; margin:0 auto"><img style="width:100%;" src="./assets/meggy.jpeg" alt="Meggy"></div>
             <h1 class="cover-heading">Megan Black</h1>
             <p class="lead">Digital Marketer, Product Visionary</a><br>Communications c/o 2018 @ USC</p>
-            <a href="mailto:meganbla@usc.edu"><img src="./assets/mail.png"></a>
+            <a href="mailto:meganbla@usc.edu"><img src="./assets/mail.png" class="Pic"></a>
```

Comment:
don't you want to animate the picture not the email link

Reply:
ugh